server-side/node-mssql-crud/src/controllers/warden.controller.js

```diff
@@ -22,4 +22,27 @@ export const gatepassApproveOrReject = async (req, res) => {
             res.status(500);
             res.send(error.message);
         }
-}
\ No newline at end of file
+}
+export const cancelApprovedGatepass= async (req,res)=>{
+    try {
+        const d = new Date();
+        let applied_date = d.getFullYear() + "-" + d.getMonth() + "-" + d.getDate();
+        const id = req.user.data.user_id;
+        const {req_id,comments}= req.body;
+        console.log(req_id);
+        console.log(comments);
+        const pool = await getConnection();
+        const result = await pool
+          .request()
+          .input("user_id", sql.VarChar, id)
+          .input("req_id",sql.Int,req_id)
+          .input("comments",sql.VarChar,comments)
+          .input("date",sql.VarChar,applied_date)
+          .input("time",sql.Time,d)
```

Comment:
what is this time please show a screenshot of the database by today as it is not working in my system.